.github/workflows/autoformat.yml

```diff
@@ -40,7 +40,7 @@ jobs:
             const autoformat = require('./.github/workflows/autoformat.js');
             const { comment } = context.payload;
             const pullInfo = await autoformat.getPullInformation(context, github);
-            const isMaintainer = autoformat.isMlflowMaintainer(comment.author_association) && autoformat.isMlflowMaintainer(pullInfo.author_association);
+            const isMaintainer = autoformat.isMlflowMaintainer(comment.author_association) && (autoformat.isMlflowMaintainer(pullInfo.author_association || pullInfo.author === "mlflow-automation"));
```

Comment:
`mlflow-automation` is a member in the mlflow organization. `autoformat.isMlflowMaintainer(pullInfo.author_association` should return `true`.

Reply:
I tested this:

https://github.com/mlflow/mlflow/pull/12808#issuecomment-2260304464

Reply:
Oh true, the failure I faced was because of Node thing, just got confused with the other one...